src/malcolmWidgets/textInput/WidgetTextInput.stories.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { action } from '@storybook/addon-actions';
+
+import ContainedTextInput from './WidgetTextInput.stories.container';
+
+const checkbox = (value, pending, units) => (
```

Comment:
I suspect this was copy and pasted as it probably shouldn't be called `checkbox`